### DISCUSSION TITLE
Dynmap

### DISCUSSION THREAD

**Discussion by WouterDeBanaan** [ServerListPlus/issues#3] [Ideas]
Add a direct dynmap button that will direct you to the server specified dynmap

**Comment by Brend-Smits** [comment]
In modded minecraft, it is not very common to see Dynmap being used. However, I think it would still be nice to have this option.

**Reply by MaxKorlaar** [reply]
Maybe custom URLs/links, such as with the custom servers

**Reply by Brend-Smits** [reply]
Yes, I was thinking of making this more general and allowing servers to add up to 2 or 3 "Call to Actions". These call to actions can be chosen in a select view on the frontend with a pre-set styled buttons/text. Example of a call to action:
"Join our Discord" - Links to Discord 
"Join our Forums" - Links to a website's forum
"Visit our Website" - Links to a website
"Check out our Launcher" - Link to launcher page to inform users about the launcher.

Things like that. This way, we don't have to write a custom implementation for every single extension that server owners could want and we can maintain the same branding across the website.
